cockroachcloud/authorization.md

```diff
@@ -1,44 +1,165 @@
+This correspondence lies in the SQL role name, which must be in the format `sso_{email_name}`. Replace '(email_name}' with the portion of the user's email address before `@`. For example, the SQL role name of a user with the email address `docs@cockroachlabs.com`  is `sso_docs`. If the role is not set up correctly, `ccloud` prompts you to create or add it. Only an admin can manage users/roles.
+
+## CockroachDB Cloud Authorization Frequently Asked Questions (FAQ)
+
+### What is the default role assigned to users as they’re added to a CockroachDB Cloud organization? What entitlements does that role include?
+
+Org Member is the default and only role assignable to new users as they are added to a CockroachDB Cloud organization. This role has most minimum entitlements across all the available roles, including just the ability to view the list of available clusters and high-level organization information like ID, Name, Label etc. 
+
+### What is the minimum access role that can be granted on a cluster?
+
+Cluster Developer is the minimum access role within the roles that could be assigned to a user on a cluster. It allows users to view the details of the target cluster(s) as well as change the IP allowlisting configuration for those cluster(s).
+
+### What roles are assigned to the user that creates a CockroachDB Cloud organization and thus becomes the first and only user in that organization?
+
+Org Member, Org Admin, Cluster Admin and Billing Coordinator [to be added soon] are assigned to the first and only user in a CockroachDB Cloud organization. This is done to allow the user perform all possible actions to create & manage clusters, configure billing & view usage etc. 
+
+Once the initial user has added more users to the CockroachDB Cloud organization, it is possible to assign Cluster Admin and Billing Coordinator roles to one or more of those users and optionally remove those roles from the initial user. Latter operation is allowed to satisfy enterprise security standards as some companies like to differentiate between product admins who can manage users, roles, and authentication configuration in a product, and the admins who are allowed to provision core resources in that product (like creating and managing clusters in CockroachDB Cloud).
+
+### Is it possible to assign more than 1 role to a user in a CockroachDB Cloud organization?
+
+Yes, it is possible to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB Cloud organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Admin, Cluster Admin and Billing Coordinator [to be added soon] roles when they create the CockroachDB Cloud organization. 
```

Comment:
```suggestion
Yes, it is possible to grant more than one role to a member. The default minimum access role Org Member is always granted to every member as long as they’re a part of the {{ site.data.products.db }} organization. Each additional granted role adds to the overall entitlements of a user. For example, the organization's creator is automatically granted the Org Member, Org Admin, Cluster Admin and Billing Coordinator [to be added soon] roles. 
```
In general, I think you "grant" or "revoke" roles rather than "assign" or "unassign" roles. WDYT?

Reply:
This usage is common throughout the docs, so punting on changing it here.